Gemfile.lock

```diff
@@ -245,33 +247,34 @@ GEM
     method_source (1.0.0)
     mime-types (3.3.1)
       mime-types-data (~> 3.2015)
-    mime-types-data (3.2019.1009)
-    mimemagic (0.3.4)
+    mime-types-data (3.2020.0512)
+    mimemagic (0.3.5)
     mini_magick (4.10.1)
     mini_mime (1.0.2)
     mini_portile2 (2.4.0)
     minitest (5.14.1)
     msgpack (1.3.3)
     mysql2 (0.5.3)
-    net-scp (2.0.0)
-      net-ssh (>= 2.6.5, < 6.0.0)
-    net-ssh (5.2.0)
+    net-scp (3.0.0)
+      net-ssh (>= 2.6.5, < 7.0.0)
+    net-ssh (6.0.2)
```

Comment:
### [net-ssh](https://github.com/net-ssh/net-ssh)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 5.2.0 → 6.0.2

[_[change-log](https://github.com/net-ssh/net-ssh/blob/master/CHANGES.txt), ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **153** commits. See the full changes on [the compare page](https://github.com/net-ssh/net-ssh/compare/v5.2.0...v6.0.2).

These are the first 10 commits:
- (cf7a7d3) [Pass :password_prompt to KeyFactory.load_private_key](https://github.com/net-ssh/net-ssh/commit/cf7a7d3dfceebf3b56f693fa960eb0e85d9cd8de)
- (aa9e6bb) [Test](https://github.com/net-ssh/net-ssh/commit/aa9e6bbc64ff7c39c56508f59c48fe2ee58416bf)
- (2708166) [remove RC4 from cipher factory](https://github.com/net-ssh/net-ssh/commit/2708166f5ec03ff3e75eaa73bde70babf62a5178)
- (e427eeb) [remove RC4 from algorithm](https://github.com/net-ssh/net-ssh/commit/e427eebd2012abb4e5dec25aacd9bffdda39e6fc)
- (1bf83d8) [remove RC4 from state](https://github.com/net-ssh/net-ssh/commit/1bf83d8c664840674cf97e2c452b4282375e7f99)
- (49c3c19) [Delete RC4 test in test_cipher_factory](https://github.com/net-ssh/net-ssh/commit/49c3c191a7e7f4eab77927829a8c6d10083b9b0a)
- (933922f) [Delete RC4 test in test_algorithms](https://github.com/net-ssh/net-ssh/commit/933922ffa7e9a3497320742564dba62865e9d599)
- (c8e4881) [removing arcfour in test_packet_stream](https://github.com/net-ssh/net-ssh/commit/c8e4881c84710ff3d20da7551e17c1d1295519d3)
- (e32f66c) [fix to key_len_override](https://github.com/net-ssh/net-ssh/commit/e32f66c8fd7be8ee3b2c5ed23248f47bd3af4ab1)
- (b504c96) [fix formatting issue to test_algorithms.rb](https://github.com/net-ssh/net-ssh/commit/b504c9693e8eb42c43c09c0c8e9768b85fccca59)


</details>